ValidationLibrary.Rules/HasLicenseRule.cs

```diff
@@ -54,6 +55,14 @@ public Task<ValidationResult> IsValid(IGitHubClient client, Repository repositor
             return Task.FromResult(new ValidationResult(RuleName, HowToFix, true, DoNothing));
         }
 
+        public Dictionary<string, string> GetConfiguration()
+        {
+            return new Dictionary<string, string>
+            {
+                 { "PullRequestTitle", RuleName }
```

Comment:
This rule doesn't create a Pull Request, so `PullRequestTitle` doesn't make sense here.